app/policies/theblog/page_policy.rb

```diff
@@ -0,0 +1,11 @@
+module Theblog
+  class PagePolicy < ApplicationPolicy
```

Comment:
I'd like not to use same class for both them because policies could be changed in the future (eg editor can create posts but not pages)
So their similarity is just corner case